unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarModifiers/AvatarModifierArea.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using DCL;
+using DCL.Components;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AvatarModifierArea : BaseComponent
+{
+    [Serializable]
+    public class Model
+    {
+        // TODO: Change to TriggerArea and handle deserialization with subclasses
```

Comment:
This is related to the comment below:

> JSON deserialization doesn't detect subclasses correctly. So I set the only subclass we have right now. If/when we decide to support other areas, we will need to change this

If we add a new type of area, it won't work directly. That comment is there so that people understand what needs to be done